wagtail/images/views/images.py

```diff
@@ -90,11 +92,10 @@ def get_base_queryset(self):
             .prefetch_renditions("max-165x165")
         )
 
-        # Annotate with usage count from the ReferenceIndex if using the list layout
-        if self.layout == "list":
-            return images.annotate(
```

Comment:
Now that we add the ability to order by usage, we need to have the annotation for the grid view as well